synth.metadata

```diff
@@ -33,5 +40,86 @@
         "generator": "bazel"
       }
     }
+  ],
+  "generatedFiles": [
```

Comment:
Not 100% sure what this does, but it was called "tracking obsolete files" here: https://github.com/googleapis/python-cloud-core/pull/43